pkg/skyenv/values_windows.go

```diff
@@ -0,0 +1,13 @@
+//+build windows
+
+package skyenv
+
+const (
+	// TODO (darkrengarius): change path
+	packageSkywirePath = "/opt/skywire"
```

Comment:
We should probably change this to `C:` drive.

Reply:
this doesn't currently run on windows, and I have no machine to test it. Hence, I'm going to propagate this issue to the Windows Build PR. Thanks for the suggestion.